src/components/App.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {
+  Switch,
+  Route,
+  Redirect,
+} from 'react-router-dom';
+import './App.scss';
+import BrokersContainer from './Brokers/BrokersContainer';
+import TopicsContainer from './Topics/TopicsContainer';
+import NavConatiner from './Nav/NavConatiner';
+import PageLoader from './common/PageLoader/PageLoader';
+import Dashboard from './Dashboard/Dashboard';
+
+interface AppProps {
+  isClusterListFetched: boolean;
+  fetchClustersList: () => void;
+}
+
+const App: React.FC<AppProps> = ({
+  isClusterListFetched,
+  fetchClustersList,
+}) => {
+  React.useEffect(() => { fetchClustersList() }, [fetchClustersList]);
```

Comment:
What about reloading cluster list, I think in nearest future it will be nice to have that feature?